<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlightForm</class>
 <widget class="QWidget" name="AddFlightForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>270</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancel_but">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="add_but">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_num">
       <property name="text">
        <string>№ авиарейса: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_num">
       <property name="inputMask">
        <string>AAA-999</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_nameair">
       <property name="text">
        <string>Название авиакомпании: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_from">
       <property name="text">
        <string>Аэропорт отправления: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_from"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_to">
       <property name="text">
        <string>Аэропорт прибытия: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_to"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Дата отправления: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="line_date">
       <property name="inputMask">
        <string>99.99.9999</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_all_seats">
       <property name="text">
        <string>Количество мест всего: </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="line_all_seats">
       <property name="inputMask">
        <string>999</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_free_seats">
       <property name="text">
        <string>Количество мест свободных:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="line_free_seats">
       <property name="inputMask">
        <string>999</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="line_time">
       <property name="inputMask">
        <string>99:99</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>Время отправления: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_num</tabstop>
  <tabstop>line_name</tabstop>
  <tabstop>line_from</tabstop>
  <tabstop>line_to</tabstop>
  <tabstop>line_date</tabstop>
  <tabstop>line_time</tabstop>
  <tabstop>line_all_seats</tabstop>
  <tabstop>line_free_seats</tabstop>
  <tabstop>add_but</tabstop>
  <tabstop>cancel_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
